shared/transforms/shared_xccdf2table-stig.xslt

```diff
@@ -79,7 +114,8 @@
 			</td>
 			</xsl:when>
 			<xsl:otherwise>
-				<td><xsl:value-of select="@id"/></td> 
+				<xsl:variable name="v_id"><xsl:value-of select="@id"/></xsl:variable>
+				<td><a href="https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_8_security_technical_implementation_guide/{$v_id}"><xsl:value-of select="@id"/></a></td>
```

Comment:
This is probably not the best idea to reference external links. This generates a table that we ship in the `scap-security-guide-doc` package.

Reply:
I have removed the links since it can become a problem with other products as well.